backend/review/views.py

```diff
@@ -582,12 +666,10 @@ def instructor_for_course_reviews(request, course_code, instructor_id):
     check_instructor_id(instructor_id)
     instructor = get_object_or_404(Instructor, id=instructor_id)
 
-    topic = course.topic
-    course = course.topic.most_recent
-
     reviews = review_averages(
         Review.objects.filter(
-            review_filters_pcr, section__course__topic=topic, instructor_id=instructor_id
+            section__course__topic_id=course.topic_id,
```

Comment:
Are these changes related to the other ones or are these just general optimizations?

Reply:
Just general optimizations